portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -42,14 +47,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int count = Integer.parseInt(request.getParameter("count"));
```

Comment:
nit: this isn't the count per se.  It's the requested number of comments.  or numRequested